app/controllers/jobseekers/uploaded_job_applications/personal_details_controller.rb

```diff
@@ -0,0 +1,43 @@
+class Jobseekers::UploadedJobApplications::PersonalDetailsController < Jobseekers::JobApplications::BaseController
+  before_action :set_job_application
+
+  helper_method :job_application, :vacancy
+
+  def edit
+    @form = Jobseekers::UploadedJobApplication::PersonalDetailsForm.new(@job_application.slice(:first_name, :last_name, :email_address, :phone_number, :has_right_to_work_in_uk))
```

Comment:
This form should be in the same namespace as the controller, so we don't need this long name? In Jobseekers::UploadedJobApplications

Reply:
Yeah not so sure about that one, I couldn't get it to work anyway